File: more_exercises/02_conditional_statements/04_transport_price.py
kilometers = int(input())
daytime = input()

price_day_taxi = 0.79
price_night_taxi = 0.90
price_bus = 0.09
price_train = 0.06
total_price = 0

if daytime == "day":
    if kilometers < 20:
        total_price = 0.70 + kilometers * price_day_taxi
    elif 20 <= kilometers < 100:
        total_price = kilometers * price_bus
    elif kilometers >= 100:
        total_price = kilometers * price_train
elif daytime == "night":
    if kilometers < 20:
        total_price = 0.70 + kilometers * price_night_taxi
    elif 20 <= kilometers < 100:
        total_price = kilometers * price_bus
    elif kilometers >= 100:
        total_price = kilometers * price_train

print(f"{total_price:.2f}")

File: 05_JavaScriptBasics/03_MoreExercises/02_ConditionalStatements/04_transportPrice.js
function transportPrice(input){
    let kilometers = Number(input[0])
    let daytime = input[1]
    let priceDayTaxi = 0.79
    let priceNightTaxi = 0.90
    let priceBus = 0.09 // minumum 20km
    let priceTrain = 0.06 // minimum 100 km

    let totalPrice = 0

    if (daytime === "day"){
        if (kilometers < 20){
            totalPrice = 0.70 + kilometers * priceDayTaxi
        } else if (kilometers >= 20 && kilometers < 100){
            totalPrice = kilometers * priceBus
        } else if (kilometers >= 100){
            totalPrice = kilometers * priceTrain
        }
    } else if (daytime === "night"){
        if (kilometers < 20){
            totalPrice = 0.70 + kilometers * priceNightTaxi
        } else if (kilometers >= 20 && kilometers < 100){
            totalPrice = kilometers * priceBus
        } else if (kilometers >= 100){
            totalPrice = kilometers * priceTrain
        }
    }
    console.log(`${totalPrice.toFixed(2)}`)
}
